src/NHibernate.Driver.Oracle.Managed/NHibernate.Driver.Oracle.Managed.csproj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../build-common/NHibernate.props" />
+
+  <PropertyGroup>
+    <Description>Driver for Oracle databases to be used with NHibernate 5.</Description>
+    <PackageTags>NHibernate; Driver; ODP.Net; Oracle; ADO.Net</PackageTags>
+
+    <TargetFramework>net461</TargetFramework>
```

Comment:
Since the beta version of `Oracle.ManagedDataAccess` with .NET Core support isn't on NuGet yet, it shouldn't be added yet, but once it is, then yes, that would be an option.

I addressed the rest of the feedback as well.